website/static/css/site.css

```diff
@@ -2109,3 +2130,96 @@ span#profileFullname{
 .modal-backdrop {
     position: fixed;
 }
+
+/* Wiki additions */
+.wiki-panel {
+    border: 1px solid #ccc;
+}
+.superlist li.active {
+    background: #EFEFEF;
+    font-weight: bold;
+}
+.wiki-panel-body {
+    height: 700px;
+    overflow: auto;
+    overflow-x: hidden;
+}
+.wiki-panel-body .navbar-collapse {
+    background: #F5F5F5;
+    padding: 0px;
+}
+.wiki-panel-header {
+    background: #efefef;
+    padding: 10px;
+    font-size: 16px;
+    font-weight: 300;
+}
+
+#wiki-form {
+    padding: 10px;
+}
+
+.markdown-it-view {
+    padding: 10px;
+}
+
+.wiki-compare-view {
+    white-space: pre-wrap;
+    padding: 10px;
+}
+
+@media (max-width: 991px) {
+    .wiki-panel {
+        margin-bottom: 20px;
+    }
+}
+.panel-collapsed {
+    background: #F5F5F5;
+}
+.wiki-sidenav h4, .wiki-sidenav li>ul>.btn,  .osf-sidenav.wiki-sidenav ul>li .row{
+    padding: 8px 15px;
+    margin-top: 2px;
+}
+.wiki-sidenav h4 {
+    margin-top:20px;
+    margin-bottom: 0;
+}
+.osf-sidenav.wiki-sidenav ul>li a:hover {
+    background: transparent;
+}
+.osf-sidenav.wiki-sidenav ul>li:hover {
+    background: #f5f5f5;
+}
+.osf-sidenav.wiki-sidenav ul>li.active a {
+    line-height: 26px;
+}
+.osf-sidenav.wiki-sidenav ul>li.active, .osf-sidenav.wiki-sidenav ul>li.active:hover {
+    background: #f5f5f5;
+    border-left: 2px solid #337AB7;
+}
+.osf-sidenav.wiki-sidenav ul>li.active a {
+    border-right: none;
+    color: #337AB7;
+}
+.osf-sidenav.wiki-sidenav h4 a:hover {
+    text-decoration: none;
+}
+.osf-sidenav.wiki-sidenav ul>li a:hover {
+    text-decoration: none;
+}
+/* Changes to default icons to be consistent with the grey look of many new ui items */
+.btn-default-alt {
+    background: #EFEFEF;
+}
+.btn-success-alt {
+    background: #D9E8D9;
+}
+
+
+@media (min-width: 1601px) {
```

Comment:
Wait why is this in the entire site.css? It means this will apply to everything on the website. 

Anything below this is bootstrap's regular sizing for fixed containers, which causes the constained widths. We can reduce this but this css should be in the wiki.mako or someplace where only wiki is loading.